dashboards-observability/public/components/common/search/autocomplete.tsx

```diff
@@ -359,6 +359,7 @@ export function Autocomplete({
         {autocompleteState.isOpen &&
           autocompleteState.collections.map((collection, index) => {
             const { source, items } = collection;
+            items.filter((item, index) => { return items.indexOf(item) === index })
```

Comment:
Do we know the root cause of duplicates? 
* Is it like making two requests?
* Browser caching?
* Or the variable is not being cleared?

Reply:
Not 100% sure but there is only one request being made and the variable should only be updated once because it's in an if condition that it must be empty for it to be updated. Don't know how to check the browser caching.

Reply:
Wait, just kidding. I just checked and it looks like there are two requests being made.